Pagination: add Getting Started tutorial (#6890)
DevExpress/devextreme-documentation: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/10 Render Items.md

> The render function determines the subset of cards to be displayed and populates the array with required images and alt strings.

Reviewer: ```suggestion
The render function determines the subset of cards to be displayed and populates the array with images and alt strings.
```

> The render function determines the start and end indexes of the visible cards array. Next, the array populates with image sources and alt strings.

Reviewer 2: ```suggestion
The render function determines the subset of cards to be displayed and populates the array with required images and alt strings.
```
